### PR TITLE
Gravatarfix

### PR DESCRIPTION
**Pull request by nikogio**
Fixed loading of gravatar images

changed helmet package as a middleware to handle image loading from www.gravatar.com

### CODE REVIEW

File: itu-minitwit-node/package-lock.json
Comment (Matej-Basta) [inline itu-minitwit-node/package-lock.json:1]: please, put this into your gitignore and take it away from here. :)

File: itu-minitwit-node/src/views/timeline.jade [hunk 1]
```diff
@@ -5,6 +5,7 @@
 -   return formatted
 - }
 
+
```
Comment (Matej-Basta) [inline itu-minitwit-node/src/views/timeline.jade:8]: This is probably not necessary. :D

**Review by Matej-Basta — CHANGES_REQUESTED** [review]

File: itu-minitwit-node/package-lock.json
Comment (nikogio) [reply itu-minitwit-node/package-lock.json:1]: thanks mate! youll see a brand new pull request created for this

**Comment by nikogio** [comment]
forgot to stash package lock. created a new pull request

**Pull request closed without merging**